src/System.Management.Automation/help/MUIFileSearcher.cs

```diff
@@ -115,22 +115,9 @@ private void SearchForFiles()
 
         private string[] GetFiles(string path, string pattern)
         {
-#if UNIX
-            // On Linux, file names are case sensitive, so we need to add
-            // extra logic to select the files that match the given pattern.
-            ArrayList result = new ArrayList();
-            string[] files = Directory.GetFiles(path);
-            foreach (string filePath in files)
-            {
-                if (filePath.IndexOf(pattern, StringComparison.OrdinalIgnoreCase) >= 0)
-                {
-                    result.Add(filePath);
-                }
-            }
-            return (String[])result.ToArray(typeof(string));
-#else
-            return Directory.GetFiles(path, pattern);
-#endif
+            // On Linux, file names are case sensitive, so we need to put both string to lower case.
+            // We can not use StringComparison.OrdinalIgnoreCase here as api supports OrdinalIgnoreCase won't support pattern
+            return Directory.GetFiles(path.ToLower(), pattern.ToLower());
```

Comment:
ToLower() is not required for path, as GetFiles considers it as case insensitive.